include/envoy/config/grpc_mux.h

```diff
@@ -125,6 +125,16 @@ class GrpcMux {
                                   const std::set<std::string>& resources,
                                   SubscriptionCallbacks& callbacks,
                                   std::chrono::milliseconds init_fetch_timeout) PURE;
+
+  /**
+   * The only difference between addToWatch() and addOrUpdateWatch() is that the 'resources' here
+   * means the *extra* resources we interested in.
+   */
+  virtual Watch* addToWatch(const std::string& type_url, Watch* watch,
```

Comment:
Is it possible to split a separate PR for review?

I do not believe that this part of the code is part of the rocketmq_proxy filter, and this part of the code does not pass review in #8984 .

Reply:
> Is it possible to split a separate PR for review?
> 
> I do not believe that this code is part of the rocketmq_proxy filter, and this part of the code does not pass review in #8984 .

I would start a new PR about this part, Thank you.

Reply:
> > Is it possible to split a separate PR for review?
> > I do not believe that this code is part of the rocketmq_proxy filter, and this part of the code does not pass review in #8984 .
> 
> I would start a new PR about this part. Thank you.

#9626